index.html

```diff
@@ -368,6 +368,8 @@ <h3>Install the certificate</h3>
 
                                 <template v-if="control == 'max'">
                                     <p>Using the <a rel="noopener" target="_blank" href="https://mozilla.github.io/server-side-tls/ssl-config-generator/?hsts=no">Mozilla SSL Configuration Generator</a> you can build the config needed to add to your server config files. e.g. if using Apache2 you'll need to either change your .htaccess or apache.conf file.</p>
+                                    <p>If you used <a rel="noopener" target="_blank" href="https://certbot.eff.org/">Certbot</a> or anothe ACME client and didn't chose automatic webserver configuration, change the paths in your webserver config to those printed by Certbot or your ACME client. Certbot usually stores certificates in <code>/etc/letsencrypt/live/</code>.</p>
+                                    <p>If you obtained your certificate in another way, upload it to your server. It's common to store certificates in <code>/etc/ssl/</code>. Make sure that your private key isn't accessible publicly and can't be read by other users. Then change the paths in your webserver config to point to it.</p>
                                     <p>Then upload the certificates to the paths shown in the config, e.g. <code>/path/to/signed_certificate</code></p>
```

Comment:
New lines about certbot etc. looks great, maybe this line under it needs removing now?